README_MULTIVAC.md

```diff
@@ -0,0 +1,77 @@
+and word case (CAS) from the Stanford natural language processing toolkit as
+inputs in the context reader. In addition, the QG-Net input includes a
+binary-valued indicator to indicate whether a word is the "answer." Finally,
+the question generator generates diverse question text word-by-word given all
+context word representations.
+
+## Query Output
+The resulting queries are then applied in order to MULTIVAC’s Markov logic
+network (MLN) ontology and the resulting answers are compared with the "answer"
+extracted directly from the source data. Good matches, in the sense of high
+scores for semantic completeness, lexical similarity, and syntactical
+similarity will indicate good performance. These measures will be assessed
+using approaches including but not necessarily limited to latent semantic
+analysis and minimal edit distance metrics. With QG-Net, we generated 2,804
+queries based on epidemiology research abstracts scraped and processed as
+earlier steps in MULTIVAC.
+
+## Dependencies
+To run QG-Net, there are a number of dependencies above and beyond those required for MULTIVAC. Primary among them is the need for a GPU-based system (currently tested against a Nvidia Quadro Pro-4000 card) for computation of the RNN. In addition to the GPU, the following are core requirements:
+- GPU for computation
+- Python 3.6
+- pytorch-0.4.1
+- OpenNMT-py (note, the version of OpenNMT-py in the original QG-Net is necessary for execution; not tested with MIT's version of OpenNMT)
+- [Stanford CoreNLP server](https://stanfordnlp.github.io/CoreNLP/#download) (optional)
+- torchtext-0.1.1
+
+## Directions to run QG-Net as a stand-alone application for MULTIVAC
+1.  Clone the [QG-Net repository on GallupGovt](https://github.com/GallupGovt/qgnet) to implement QG-Net within the MULTIVAC system.
+2.  The MULTIVAC instantiation of QG-Net uses a domain-adapted GloVe model. This should be a text file outputted from an earlier step of the MULTIVAC system. It needs to be updated, within the QG-Net repo, in the following file: `./OpenNMT-py/opts.py`.
+3.  Run the download script `download_QG-Net.sh` in the top-level directory in a terminal window to download a pre-trained QG-Net model. You may need to modify the shell script to fit your needs. This will create the QG-Net model needed for modeling.
+4. The Python script `preprocessing_pdf.py` in the `./test/` folder takes the epidemiological research articles as inputs in the form of JSON files. Using the JSON files, the Python script generates the `input.txt` file which includes the "context" and "answers" needed for input into the QG-Net system. The `input.txt` file is saved in the `./test/input.for.test/` directory.
+4.  Run the script in the `./test/` folder; first `cd test/`, then run the shell script `qg_reproduce_LS.sh`. This takes the file `input.txt` and generates questions using the QG-Net algorithm. The final output is saved in the `./test/` folder.
```

Comment:
Maybe we should state what the name of the final output file is saved as, for example, a file with the prefix output_questions

Reply:
Can do!